push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/fcm/ConfigurableFCMSender.java

```diff
@@ -36,11 +36,15 @@ public ConfigurableFCMSender(String key) {
 
 	@Override
 	protected HttpURLConnection getConnection(String url) throws IOException {
-		
+
 		// let's see if there is a different URL we should post to (e.g. load/stress testing)
 		final String fcmURL = ConfigurationUtils.tryGetProperty(CUSTOM_AEROGEAR_FCM_PUSH_HOST, FCM_ENDPOINT_HOST);
-		
+
```

Comment:
please refrain from unneeded white space changes

Reply:
This is the replacement of an unneeded tabbed line with a clean newline.

Reply:
OK